edenai_apis/llmengine/clients/completion.py

```
@@ -60,6 +60,11 @@ async def aresponses(
             "Not implemented. Please implement this method in a subclass."
         )
 
+    async def aanthropic_messages(self, model: str = None, **kwargs):
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)